src/webpage.js

```diff
@@ -0,0 +1,101 @@
+var express = require('express');
+var app = express();
+
+/**
+ * @api {use} Error Handling
+ * @apiName Error
+ * @apiGroup Test
+ */
+app.use(function(err,req,res,next){
+    res.status(500).send('Error');
+    next();
+});
+
+/**
+ * @api {get} / Request Test Route 
+ * @apiName Get Homepage
+ * @apiGroup Test
+ * 
+ * @apiSuccess {String} homepage gets homepage
+ * 
+ * @apiSuccessExample Success URL
+ * {
+ *  URL: "/"
+ * }
+ */
+app.get('/', function(req,res){
+    res.sendFile(__dirname + '/src/index.html');
+});
+
+/**
+ * @api {get} /contact Request Test Route
+ * @apiName Get Contact
+ * @apiGroup Test
+ * 
+ * @apiSuccess {String} contact gets contact
+ * 
+ * @apiSuccessExample Success URL
+ * {
+ *  URL: "/contact"
+ * }
+ */
+app.get('/contact', function(req,res){
+    res.send('This is the contact page');
+});
+
+/**
+ * @api {get} /contact/:name Request Test Route
+ * @apiName Contact Page Error Handling
+ * @apiGroup Test
+ * 
+ * @apiSuccess {String} contact error handling
+ * 
+ * @apiSuccessExample Success URL
+ * {
+ * URL: "/contact/name"
+ * }
+ */
+app.get('/contact/:name', function (req,res){
+    try {
+        re.send('dejid');
```

Comment:
not the best way to emulate error. I would rather do a throw Error() if :name does not equals something. Otherwise you're just embedding a execution exception in a code, which is a bad habit to have